nibabel/testing/__init__.py

```diff
@@ -164,3 +166,9 @@ def __init__(self, *args, **kwargs):
         warnings.warn('catch_warn_reset is deprecated and will be removed in '
                       'nibabel v3.0; use nibabel.testing.clear_and_catch_warnings.',
                       FutureWarning)
+
+
+def skipif_no_slow_tests(func):
```

Comment:
My thought was: it's good to have the very clear headers with no parameters, so the options stay very limited and no mistakes can be made. The generic function above could be used as the base for each, and could be written if/when needed...

That was my thought at least.